docs/getting-started/releases.md

```diff
@@ -4,97 +4,17 @@ category: Getting Started
 order: 8
 ---
 
-## Releases
```

Comment:
This deleted block seems to contain some important info, should we delete them all? What do you think @HerrTopi, do we need any of this?

Reply:
As I know most of the functionality was deleted. Also its a bit too step-by-step for me, when someone wants to contribute we can assume that they know Git commands. But I'm open to any suggestion :)

Reply:
I'd say we should keep the info that is still valid. Even the step-by-step info can be useful when a junior joins the team :)

Reply:
hmm okay, I bought this argument, I'll rewrite it :)

Reply:
I've added it back